rio_tiler/io/stac.py

```diff
@@ -100,8 +108,8 @@ class STACReader(MultiBaseReader):
     ----------
     filepath: str
         STAC Item path, URL or S3 URL.
-    item: Dict, optional
-        STAC Item dict.
+    item: pystac.Item, optional
```

Comment:
This can be `dict` or `pystac.Item`, right? Because of the `converter=to_stac_item`?